### PR TITLE
Use an alias to access destiny-icons

### PR DESCRIPTION
**Pull request by bhollis**
This makes it a bit easier to reference stuff out of destiny-icons.

### CODE REVIEW

**Comment by bhollis** [comment]
For some reason eslint/tslint isn't running in my VSCode anymore...

**Comment by delphiactual** [comment]
This works locally for me after adding the tsconfig, not sure what is wrong on TravisCI

File: src/app/item-popup/ItemPopupContainer.scss [hunk 1]
```diff
@@ -239,13 +239,13 @@
...
-  background-image: url('../../../destiny-icons/general/ammo_primary.svg');
+  background-image: url('~destiny-icons/general/ammo_primary.svg');
 }
 .ammo-special {
-  background-image: url('../../../destiny-icons/general/ammo_special.svg');
+  background-image: url('~destiny-icons/general/ammo_special.svg');
 }
 .ammo-heavy {
-  background-image: url('../../../destiny-icons//general/ammo_heavy.svg');
+  background-image: url('~destiny-icons//general/ammo_heavy.svg');
```
Comment (delphiactual) [inline src/app/item-popup/ItemPopupContainer.scss]:
```suggestion
  background-image: url('~destiny-icons/general/ammo_heavy.svg');
```

Comment (sundevour) [reply src/app/item-popup/ItemPopupContainer.scss]: ooh. hope that's it. it linted and ran fine for me locally after bhollis's fix before your tsconfig update so shrug

**Comment by bhollis** [comment]
I wonder if the eslint cache is messing with it?

**Comment by delphiactual** [comment]
Maybe.. we can disable it for this build...

**Comment by delphiactual** [comment]
@bhollis well that was it :face_with_head_bandage:

File: build/travis_build.sh [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 set -o pipefail
 
-find . -type d -name "node_modules" -prune -o -type f -iregex '.*.ts\|.*.js\|.*.tsx\|.*.jsx' -exec ./build/set-mtime-to-md5.sh {} \; && echo Set mtime to MD5 complete!
+#find . -type d -name "node_modules" -prune -o -type f -iregex '.*.ts\|.*.js\|.*.tsx\|.*.jsx' -exec ./build/set-mtime-to-md5.sh {} \; && echo Set mtime to MD5 complete!
```
Comment (delphiactual) [inline build/travis_build.sh]:
```suggestion
find . -type d -name "node_modules" -prune -o -type f -iregex '.*.ts\|.*.js\|.*.tsx\|.*.jsx' -exec ./build/set-mtime-to-md5.sh {} \; && echo Set mtime to MD5 complete!
```

**Comment by delphiactual** [comment]
Easier way, delete the cache file @ Travis

https://travis-ci.org/DestinyItemManager/DIM/caches

**Pull request merged**